### PR TITLE
Multiple user problem solved 

### PR DESCRIPTION
**Pull request by Dhroov7**

### CODE REVIEW

**Comment by Dhroov7** [comment]
@championswimmer Please review!

File: src/passport/strategies/user/strategy-facebook.js [hunk 1]
```diff
@@ -33,10 +33,11 @@ module.exports = new FacebookStrategy({
...
     try{
         if(oldUser) {
-            const fbaccount = await  models.UserFacebook.findOne({where: {id: profileJson.id}})
+            const fbaccount = await  models.UserFacebook.findAll({where: {id: profileJson.id}})
+            let oldIds = fbaccount.map((uf) => uf.id).join(',')
 
-            if (fbaccount) {
-                throw new Error('Your Facebook account is already linked with codingblocks account Id: ' + fbaccount.dataValues.userId)
+            if (fbaccount.length > 0) {
+                throw new Error('Your Facebook account is already linked with codingblocks account Id: ' + oldIds)
```
Comment (championswimmer) [inline src/passport/strategies/user/strategy-facebook.js]: the facebook account will be linked to at least 1 
that's the one the user is already logged in to

File: src/passport/strategies/user/strategy-github.js [hunk 2]
```diff
@@ -27,9 +27,10 @@ module.exports = new GithubStrategy({
     try{
         if(oldUser){
             debug('User exists, is connecting Github account')
-            const ghaccount =  await models.UserGithub.findOne({where: {id: profileJson.id}})
-            if (ghaccount) {
-                throw new Error('Your Github account is already linked with codingblocks account Id: ' + ghaccount.dataValues.userId)
+            const ghaccount =  await models.UserGithub.findAll({where: {id: profileJson.id}})
```
Comment (championswimmer) [inline src/passport/strategies/user/strategy-github.js]: please do not make any changes to the if branch where oldUser exists. that side of the if condition is completely correct.

**Comment by Dhroov7** [comment]
@championswimmer Done with the changes!

**Pull request merged**